src/parcsr_ls/par_lr_restr_device.c

```diff
@@ -0,0 +1,411 @@
+/******************************************************************************
+ * Copyright 1998-2019 Lawrence Livermore National Security, LLC and other
+ * HYPRE Project Developers. See the top-level COPYRIGHT file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ ******************************************************************************/
+
+#include "_hypre_parcsr_ls.h"
+#include "_hypre_utilities.hpp"
+
+#if defined(HYPRE_USING_CUDA) || defined(HYPRE_USING_HIP)
+
+__global__ void hypre_BoomerAMGBuildRestrNeumannAIR_assembleRdiag( HYPRE_Int nr_of_rows, HYPRE_Int *Fmap, HYPRE_Int *Cmap, HYPRE_Int *Z_diag_i, HYPRE_Int *Z_diag_j, HYPRE_Complex *Z_diag_a, HYPRE_Int *R_diag_i, HYPRE_Int *R_diag_j, HYPRE_Complex *R_diag_a);
+
+/*---------------------------------------------------------------------------
+ * hypre_BoomerAMGBuildRestrNeumannAIR
+ *--------------------------------------------------------------------------*/
+HYPRE_Int
+hypre_MatvecCommPkgCreateCustom ( hypre_ParCSRMatrix *A )
```

Comment:
I did not find differences between this one and `hypre_MatvecCommPkgCreate`. Maybe I missed something?

Reply:
This was a holdover from some debugging. I'll remove the "Custom" routine and replace its use with the regular one. Thanks.